Formula/h2.rb

```diff
@@ -21,14 +21,6 @@ def install
     # Remove windows files
     rm_f Dir["bin/*.bat"]
 
-    # As of 1.4.190, the script contains \r\n line endings,
-    # causing it to fail on macOS. This is a workaround until
-    # upstream publishes a fix.
-    #
-    # https://github.com/h2database/h2database/issues/218
-    h2_script = File.read("bin/h2.sh").gsub("\r\n", "\n")
-    File.open("bin/h2.sh", "w") { |f| f.write h2_script }
-
     # Fix the permissions on the script
     chmod 0755, "bin/h2.sh"
```

Comment:
This one too

Reply:
I  checked the permission of the file, looks like we still need 0755 for it?

Reply:
Anyway, let me see if it can be removed or not.

Reply:
If it does that should be reported upstream.

Reply:
sure, I will raise this up to the upstream.

Reply:
https://github.com/h2database/h2database/issues/3254